### PR TITLE
fix discordjs starter

### PR DESCRIPTION
**Pull request by PantheraRed**
Bumped [discord.js](https://discord.js.org) version from `12.5.1` to `12.5.3`, added typescript & 
fixed [README](https://github.com/railwayapp/starters/blob/master/README.md).

### CODE REVIEW

File: examples/discordjs/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Client, Message } from 'discord.js';
+const { DISCORD_TOKEN } = process.env; // The TOKEN from railway
```
Comment (1chiSensei) [inline examples/discordjs/src/index.ts:2]: Omit this, discord.js auto detects this var.

File: examples/discordjs/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+client.on('message', (msg: Message) => {
+	if (msg.content === 'ping') {
+		msg.reply('Pong 🏓');
+	} else if (msg.content === 'hello') {
+		msg.reply('Choo choo! 🚅');
+	}
+});
+
+client.login(DISCORD_TOKEN);
```
Comment (1chiSensei) [inline examples/discordjs/src/index.ts:18]: Omit this too.

**Review by 1chiSensei — CHANGES_REQUESTED** [review]

File: examples/discordjs/src/index.ts (hunk 2 above)
Comment (1chiSensei) [reply examples/discordjs/src/index.ts:18]: Replace it with `client.login();`.

Comment (PantheraRed) [reply examples/discordjs/src/index.ts:18]: Ya I wasn't sure this was part of v13 or v12 so I decided to keep it that way. However I've went ahead & confirmed that it is able to check token from env. Thanks for your feedback. I'll try to change this as soon as possible. 👍

File: examples/discordjs/README.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
 ---
-title: Discord bot
-description: A Discord bot written in TypeScript
+title: Discord JavaScript Bot
```
Comment (1chiSensei) [inline examples/discordjs/README.md:2]:
```suggestion
title: Discord TypeScript Bot
```

File: examples/discordjs/README.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+description: A Discord Bot written in TypeScript
 tags:
   - discord.js
   - typescript
 ---
 
 # Discord.js Example
 
-This example starts a Discord bot using [discord.js](https://discord.js.org/#/).
+A basic Discord Bot using [discord.js](https://discord.js.org/#/).
```
Comment (1chiSensei) [inline examples/discordjs/README.md:11]:
```suggestion
A basic Discord Bot using [discord.js](https://discord.js.org/).
```

Why is this hash needed?

**Review by 1chiSensei — CHANGES_REQUESTED** [review]

File: examples/discordjs/README.md (hunk 3 above)
Comment (1chiSensei) [reply examples/discordjs/README.md:2]: @PantheraRed could you change this?

**Comment by PantheraRed** [comment]
I think this pull request is complicating the example for beginners. It would be best to keep it simple & make a separate catagory of this. I'll be closing this pull request & will keep the suggested changes above in mind for future reference. 👍

**Pull request closed without merging**